Client/Experiments/Messaging/MessagingHelper.swift

```diff
@@ -0,0 +1,101 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+import MozillaAppServices
+import Shared
+
+protocol MessagingHelperProtocol { }
+
+extension MessagingHelperProtocol {
+    var messagingHelper: MessagingHelper {
+        return MessagingHelper.shared
+    }
+}
+
+/// The Message Helper is responsible for preparing fetched messages to appear in a UI surface.
+/// It should do all operations on a message and return only valid, eligible, non-expired messages FOR the associated surface.
+class MessagingHelper: Loggable {
+    
+    // MARK: - Properties
+    
+    static let shared = MessagingHelper()
+    
+    init() {
+        
+    }
+    
+    // MARK: - Public helpers
+    
+    /// JEXLs are more accurately evaluated when given certain details about the app on device.
+    /// There is a limited amount of context you can give. See:
+    /// - https://experimenter.info/mobile-messaging/#list-of-attributes
+    func createAdditionalContext() -> [String: Any] {
```

Comment:
```suggestion
    func createAdditionalDeviceContext() -> [String: Any] {
```
??? Maybe? What addinitonal context